src/main/java/org/jenkinsci/plugins/api/BitbucketApiService.java

```diff
@@ -29,17 +29,25 @@
 
     private OAuthService service;
 
+    private String[] roles;
+
     public BitbucketApiService(String apiKey, String apiSecret) {
         this(apiKey, apiSecret, null);
     }
 
     public BitbucketApiService(String apiKey, String apiSecret, String callback) {
+        this(apiKey, apiSecret, callback,null);
```

Comment:
This should pass the default set of roles or you could end up with a NPE in your for loop below.